packages/amplication-server/src/core/account/account.service.ts

```diff
@@ -87,4 +87,13 @@ export class AccountService {
       where: { id: accountId },
     });
   }
+
+  setLastActive(accountId: string, lastActive: Date): Promise<Account> {
```

Comment:
I think it is better to use this method on the User service, and send the entire user object to the function instead of accountId

UserService.SetLastActivity(user:User)

This way, you can just call the function without the need to know how to convert a user to the account, so the interface will be easier to understand